dashboard/forms.py

```diff
@@ -0,0 +1,21 @@
+from email.policy import default
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+
+def validateemail(value):
+    """checking if given email alreday taken or not"""
+    if User.objects.filter(email = value).exists():
+        raise ValidationError((f"{value} is taken!"),params = {'value':value})
+
+class UserCreationForm(UserCreationForm):
```

Comment:
you are overriding a variable from external scope. give this class a separate name

Reply:
made the change -rename UserCreationForm to NewUserCreationForm